code/modules/mob/mob.dm

```diff
@@ -1009,7 +1009,8 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 
 // this function displays the station time in the status panel
 /mob/proc/show_stat_station_time()
-	stat(null, "Round Time: [worldtime2text()]") // AA TODO: Make this do "Game Time" and "Round Time" with the ROUND_TIME macro
+	stat(null, "Server Uptime: [worldtime2text()]")
+	stat(null, "Round Time: [ROUND_TIME ? time2text(ROUND_TIME, "hh:mm:ss") : "N/A"]")
 	stat(null, "Station Time: [station_time_timestamp()]")
```

Comment:
This would be because I'm in BST currently rather than GMT, but the 'Server Uptime' on my end is underflowing to 23 hours at roundstart:
![image](https://user-images.githubusercontent.com/57483089/127216538-c0f07b26-a0c2-4c5f-b643-be48dee6df17.png)
Might not need to be fixed since I know the actual server is on ISO 8601/UTC, but I figured I'd point this out anyway.

Reply:
Thats because of daylight savings. BYOND breaks with that enabled. It should work fine in prod.